tests/fork_func/check_exec.c

```diff
@@ -0,0 +1,39 @@
+#include <unistd.h>
+
+#include "fork_func.h"
+#include "warnp.h"
+
+#include "check.h"
+
+static int
+func_exec(void * cookie)
+{
+
+	(void)cookie; /* UNUSED */
+
+	if (execlp("true", "", NULL))
```

Comment:
Is there a reason you want to invoke `true(1)` with an argv0 of ""?

Reply:
nope, fixed.